pages/about/security-information.md

```diff
@@ -0,0 +1,24 @@
+---
+permalink: /about/security-information/
```

Comment:
Update to /about/security here too.

Reply:
I changed the page to name to just `security` and updated redirects.